docs/api-reference/next/image.md

```diff
@@ -47,13 +47,13 @@ When using an external URL, you must add it to
 
 ### width
 
-The width of the image, in pixels. Must be an integer without a unit.
+The desired width of the rendered image, in pixels. Must be an integer without a unit.
```

Comment:
Should we say this? In several of the layouts, the width provided to the width attribute is specifically not the desired pixel width of the image. It's the intrinsic width of the src image before responsive resizing.

Reply:
Yeah good point, we need to explain different layouts here 😮‍💨